### DISCUSSION TITLE
export sky view in svg format

### DISCUSSION THREAD

**Discussion by AnnabelleVH** [Stellarium/stellarium#3094] [Ideas]
Hi,

Thanks for developping this wonderful tool, and congrats!

It seems that images are created using svg format with inkscape and are rendered in png.
Is that possible to export the capture in svg format (it isn't listed in the options output file format, or I was not able to find it)?
That would be really nice, especially with different kind of information on separeted layers (constellations names, constellations lines, art...).

Cheers,
Annabelle

**Comment by gzotti** [comment]
While parts of the design (e.g. buttons) are created with InkScape, in no place we document any use of SVG during runtime simply because we use OpenGL to use and produce raster images. If it would be easy to create SVG or EPS vector output, we would offer it. It is not easy.

**Reply by AnnabelleVH** [reply]
Sorry for my mistake (indeed I do associate InkScape with svg format). Thanks for the quick response and congrats for the amazing work!
